ui/page/transaction/transactions_page.go

```diff
@@ -209,99 +276,159 @@ func (pg *TransactionsPage) loadTransactions(offset, pageSize int32) ([]*sharedW
+						}
 
-												gtx.Constraints.Min.X = gtx.Constraints.Max.X
-												separator := pg.Theme.Separator()
-												return layout.E.Layout(gtx, func(gtx C) D {
-													// Show bottom divider for all rows except last
-													return layout.Inset{Left: values.MarginPadding56}.Layout(gtx, separator.Layout)
-												})
-											}),
-										)
+						wallTxs := pg.scroll.FetchedData()
+						return pg.transactionList.Layout(gtx, len(wallTxs), func(gtx C, index int) D {
+							tx, wal := pg.txAndWallet(wallTxs[index])
+							return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+								layout.Rigid(func(gtx C) D {
+									isHiddenAssetsInfo := true
+									if pg.selectedWallet == nil {
+										isHiddenAssetsInfo = !isHiddenAssetsInfo
+									}
+									return components.LayoutTransactionRow(gtx, pg.Load, wal, tx, isHiddenAssetsInfo)
+								}),
+								layout.Rigid(func(gtx C) D {
+									// No divider for last row
+									if index == len(wallTxs)-1 {
+										return layout.Dimensions{}
+									}
+
+									gtx.Constraints.Min.X = gtx.Constraints.Max.X
+									separator := pg.Theme.Separator()
+									return layout.E.Layout(gtx, func(gtx C) D {
+										// Show bottom divider for all rows except last
+										return layout.Inset{Left: values.MarginPadding56}.Layout(gtx, separator.Layout)
 									})
-								})
-							})
+								}),
+							)
 						})
 					})
-				}),
-				layout.Expanded(func(gtx C) D {
-					return pg.txTypeDropDown.Layout(gtx, 0, true)
-				}),
-			)
+				})
+			})
 		})
-	})
+	})}
+
+	// disabled overlay
+	if pg.walletNotReady() && pg.isHomepageLayout {
+		layoutContents = append(layoutContents, layout.Stacked(func(gtx C) D {
+			gtx = gtx.Disabled()
+			overlayColor := pg.Theme.Color.Gray3
+			overlayColor.A = 220
+			gtx.Constraints.Min.X = gtx.Constraints.Max.X
+			gtx.Constraints.Min.Y = gtx.Constraints.Max.Y - gtx.Dp(values.MarginPadding60)
+			cryptomaterial.FillMax(gtx, overlayColor, 10)
+
+			lbl := pg.Theme.Label(values.TextSize20, values.String(values.StrFunctionUnavailable))
+			lbl.Font.Weight = font.SemiBold
+			lbl.Color = pg.Theme.Color.PageNavText
+			return cryptomaterial.CentralizeWidget(gtx, lbl.Layout)
+		}))
+	}
+
+	return layout.Stack{Alignment: layout.NW}.Layout(gtx, layoutContents...)
+}
 
-	items := []layout.FlexChild{layout.Rigid(pg.pageTitle)}
-	if pg.WL.SelectedWallet.Wallet.GetAssetType() == utils.DCRWalletAsset {
-		// Layouts only supportted by DCR
-		line := pg.Theme.Separator()
-		line.Color = pg.Theme.Color.Gray3
-		items = append(items, layout.Rigid(line.Layout))
-		items = append(items, layout.Rigid(pg.sectionNavTab))
+func (pg *TransactionsPage) layoutDesktop(gtx C) D {
+	items := []layout.FlexChild{}
+	if pg.selectedWallet == nil || pg.selectedWallet.GetAssetType() == utils.DCRWalletAsset {
+		// Only show tx category navigation txCategoryTab for DCR wallets.
+		items = append(items, layout.Rigid(pg.txCategoriesNav))
+	}
```

Comment:
Uhm, not really. Also show the tab if there are dcr wallets but no wallet is selected at the moment. Only hide it if there are no dcr wallets at all or the currently selected wallet is not a dcr wallet.

`pg.selectedWallet` may be nil but it doesn't mean that a dcr wallet exists and if a dcr wallet does not exist, no need to show the stake tab.